common/components/ui/Modal.jsx

```diff
@@ -18,6 +17,8 @@ type Props = {
       | 'warning'
       | 'danger'
       | 'link',
+    disabled?: boolean,
+    // $FlowFixMe - Why the fuck doesn't this like onClick?
```

Comment:
If anyone could help me out with this, it doesn't like the onClicks I provided in ConfirmationModal.jsx and the error is really opaque, just saying my buttons don't match the button object.